cmd/bee/cmd/start.go

```diff
@@ -346,6 +347,22 @@ func buildBeeNode(ctx context.Context, c *command, cmd *cobra.Command, logger lo
 
 	return b, err
 }
+func (c *command) CheckMissingFlag(cmd *cobra.Command, args []string) error {
+	if err := c.initConfig(); err != nil {
+		return err
+	}
+	var missingParams []string
+	for _, v := range c.config.AllKeys() {
+		if f := cmd.Flags().Lookup(v); f == nil {
```

Comment:
Since you don't need the value of `f`, the following will do: `cmd.Flags().Lookup(v) == nil`.